config/reports.json

```diff
@@ -8,51 +8,59 @@
 		"chrome-ux-report"
 	],
 	"_featured": [
-		"state-of-javascript"
+		"state-of-the-web"
 	],
 	"_metrics": {
 		"bytesCss": {
 			"name": "CSS Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all external stylesheets requested by the page. An external stylesheets is identified as a resource with the `css` file extension or a MIME type containing `css`."
 		},
 		"bytesFont": {
 			"name": "Font Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all fonts requested by the page. A font is identified as a resource with the `eot`, `ttf`, `woff`, `woff2`, or `otf` file extensions or a MIME type containing `font`."
 		},
 		"bytesHtml": {
 			"name": "HTML Bytes",
-			"type": "KB"
+			"type": "KB",
+			"description": "The sum of transfer size kilobytes of all HTML documents requested by the page. An HTML document is identified as a resource with the `html` or `html` file extensions or a MIME type containing `html`."
```

Comment:
Curious, where do we slot text/plain? :)

Reply:
That would be categorized as "text". See https://github.com/HTTPArchive/httparchive/blob/b52c778f67f7d3217e5c1e763372d4ee0a66e5a3/bulktest/batch_lib.inc#L967-L970

Not currently tracking it in these reports though.